server/utils.js

```diff
@@ -0,0 +1,29 @@
+//Probably should move more stuff into here
+
+var shortid = require("shortid")
+var logger = require("./logger")
+
+var slugify = function(str) {
```

Comment:
In fact, I already found an issue with it: "Zeros and Ones - Typescript" got sluggified as `zeros_and_ones__typescript` (note the double-underscore).